jwst/tests_nightly/general/fgs/test_guider_pipeline.py

```diff
@@ -0,0 +1,89 @@
+import os
+import pytest
+from astropy.io import fits as pf
+from jwst.pipeline.calwebb_guider import GuiderPipeline
+
+
+pytestmark = [
+    pytest.mark.usefixtures('_jail'),
+    pytest.mark.skipif(not pytest.config.getoption('bigdata'),
+                       reason='requires --bigdata')
+]
+
+
+def test_guider_pipeline1(_bigdata):
+    """
+
+    Regression test of calwebb_guider pipeline performed on ID-image data.
+
+    """
+
+    GuiderPipeline.call(_bigdata+'/fgs/test_guiderpipeline/jw88600073001_gs-id_7_image-uncal.fits',
```

Comment:
If there is any chance you would want to test under Windows, using `os.path.join` to string together (pun!) path name might be safer. But that can be future work in separate PR, if needed.

Reply:
All of these `thing + "stuff"` concatenations are going away, don't worry :smile: 
Note: I didn't bother changing any of these because I anticipated replacing all of them with something more robust.